fabric-networking-v0/src/main/java/net/fabricmc/fabric/impl/networking/PacketTypes.java

```diff
@@ -19,9 +19,9 @@
 import net.minecraft.util.Identifier;
 
 public final class PacketTypes {
-	public static final Identifier BRAND = new Identifier("minecraft:brand");
-	public static final Identifier REGISTER = new Identifier("minecraft:register");
-	public static final Identifier UNREGISTER = new Identifier("minecraft:unregister");
+	public static final Identifier BRAND = new Identifier("brand");
+	public static final Identifier REGISTER = new Identifier("register");
+	public static final Identifier UNREGISTER = new Identifier("unregister");
```

Comment:
Though not specifying the namespace will fallback to minecraft's namespace, I'd keep the namespace to be explicit